lib/scenic/configuration.rb

```diff
@@ -0,0 +1,34 @@
+module Scenic
+  class Configuration
```

Comment:
What do you think of using Rails' custom configuration (example: https://github.com/dhh/custom_configuration) instead of building up our own class?